packages/docs/src/page-configs/ui-elements/data-table/examples/CustomSlots.vue

```diff
@@ -65,8 +66,8 @@ export default defineComponent({
         id: 3,
         name: 'Clementine Bauch',
         username: 'Samantha',
-        email: 'Nathan@yesenia.net',
         address: {
+          email: 'Nathan@yesenia.net',
```

Comment:
And here. Move `email`s back to the top-most-level.

Reply:
Restored.